plugins/continuity.js

```diff
@@ -1382,6 +1382,15 @@
 					src = elements[i].currentSrc ||
 						elements[i].src ||
 						(typeof elements[i].getAttribute === "function" && elements[i].getAttribute("xlink:href"));
+	
+					// if src if not defined, look for it in css background image
+					if (!src) {
+						var bgStyle = BOOMR.window.getComputedStyle(elements[i]).getPropertyValue('background');
```

Comment:
I think we'll need a `typeof BOOMR.window.getComputedStyle === "function"` check as well, as that's only supported in IE9+ and Edge12+

Reply:
Sure, I will do that and give the test a shot.